pandas/tests/series/test_indexing.py

```diff
@@ -546,6 +546,22 @@ def test_getitem_setitem_periodindex(self):
         result[4:8] = ts[4:8]
         assert_series_equal(result, ts)
 
+    def test_getitem_with_duplicates_indices(self):
```

Comment:
parameterize this

Reply:
Sorry, parameterize this , what should I do ?

Reply:
http://pandas.pydata.org/pandas-docs/stable/contributing.html#using-pytest